e2e_tests/integration/stepDefinitions/index.ts

```diff
@@ -226,16 +221,63 @@ When('Maria confirms the existing custody status', () => {
   cy.clickButton('Continue')
 })
 
-export const assertQ1_emergency_recall = (contents: string, answer: string) => expect(contents).to.contain(`until PPCS has issued the revocation order.  ${answer}`)
-export const assertQ2_indeterminate_sentence_type = (contents: string, answer: string) => expect(contents).to.contain(`Is the offender serving a life or IPP/DPP sentence? ${answer}`)
-export const assertQ3_extended_sentence = (contents: string, answer: string) => expect(contents).to.contain(`Is the offender serving one of the following:  ${answer}`)
-export const assertQ6_custody_status = (contents: string, answer: string) => expect(contents).to.contain(`Is the offender currently in police custody or prison custody? ${answer}`)
-export const assertQ7_addresses = (contents: string, answer: string) => expect(contents).to.contain(`If the offender is in police custody, state where: ${answer}`)
-export const assertQ8_arrest_issues = (contents: string, answer: string, details: string) => {
+const data = getTestDataPerEnvironment()
+
+export const q1EmergencyRecall = (contents: string, answer: string) =>
+  expect(contents).to.contain(`until PPCS has issued the revocation order.  ${answer}`)
+export const q2IndeterminateSentenceType = (contents: string, answer: string) =>
+  expect(contents).to.contain(`Is the offender serving a life or IPP/DPP sentence? ${answer}`)
+export const q3ExtendedSentence = (contents: string, answer: string) =>
+  expect(contents).to.contain(`Is the offender serving one of the following:  ${answer}`)
+export const q4OffenderDetails = (contents: string) => {
+  expect(contents).to.contain(data.fullName)
```

Comment:
This will be more reliable if it looks for the label and the value ie `Full name: <name>`